settings.py

```diff
@@ -530,11 +530,6 @@ def JINJA_CONFIG():
 PUENTE = {
     'BASE_DIR': ROOT,
     'TEXT_DOMAIN': 'django',
```

Comment:
Right... it's still there, but I'm wondering if I should have removed it just like we did with STANDALONE_DOMAINS. It's not documented in the docs. I forget what I was thinking.